src/nls/zh-tw/strings.js

```diff
@@ -520,7 +520,7 @@ define({
     "INSTALL_EXTENSION_DROP"               : "放開 .zip 開始安裝",
     "INSTALL_EXTENSION_DROP_ERROR"         : "下列原因造成安裝或是更新失敗:",
     "INSTALL_FROM_URL"                     : "由 URL 安裝\u2026",
-    "INSTALL_EXTENSION_VALIDATING"         : "Validating\u2026",
+    "INSTALL_EXTENSION_VALIDATING"         : "檢查擴充功能中……\u2026",
```

Comment:
Remove `……`, since `\u2026` means "almost" the same thing.

Unicode Character 'HORIZONTAL ELLIPSIS' (U+2026):
http://www.fileformat.info/info/unicode/char/2026/index.htm